src/memory/mem_info.rs

```diff
@@ -44,10 +44,10 @@ impl MemoryInfo {
         let bytes_to_megabytes = |bytes: u64| (bytes / B_TO_MB) * (mem_unit as u64);
         let ratio = |x, y| ((x as f32 / y as f32) * 100.0) as u8;
 
-        let available_ram_mb = bytes_to_megabytes(sys_info.freeram.into());
```

Comment:
These conversions were intentional, although they are redundant on x86-64

In some architectures this struct uses 32-bit integers instead of 64 bit ones and these `into`s cover that

Reply:
Solved.